modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -22,26 +28,18 @@
 		to_chat(user, "You cannot send IC messages (muted).")
 		return FALSE
 	else if(!params)
-		subtle_emote = stripped_multiline_input(user, "Choose an emote to display.", "Subtle", null, MAX_MESSAGE_LEN)
+		subtle_emote = tgui_input_text(user, "Choose an emote to display.", "Subtle", null, MAX_MESSAGE_LEN, TRUE)
 		if(subtle_emote && !check_invalid(user, subtle_emote))
-			var/type = input("Is this a visible or hearable emote?") as null|anything in list("Visible", "Hearable")
-			switch(type)
-				if("Visible")
-					emote_type = EMOTE_VISIBLE
-				if("Hearable")
-					emote_type = EMOTE_AUDIBLE
-				else
-					alert("Unable to use this emote, must be either hearable or visible.")
-					return
 			subtle_message = subtle_emote
 		else
 			return FALSE
 	else
 		subtle_message = params
 		if(type_override)
 			emote_type = type_override
-	. = TRUE
+
 	if(!can_run_emote(user))
+		to_chat(user, "You can't emote at this time.", type = MESSAGE_TYPE_WARNING)
```

Comment:
Why did you do both of them differently? The other one used just a span, and this one uses the `type` parameter? Just use one and stick to it, for coherence's sake.

Reply:
Uhhh, that was one I missed, oops-